### ISSUE TITLE
Italics do not render next to (or including) subscripts

### ISSUE THREAD

**Issue by pdpinch** [mitodl/ocw-hugo-themes#501]
### Steps to Reproduce

1. Check out https://github.mit.edu/ocw-content-rc/18.s096-fall-2013 and build it
2. Take a look a the page `/projects`

### Expected Behavior

- formulas are in italics, with subscripts

<img width="790" alt="image" src="https://user-images.githubusercontent.com/430126/156658244-a791954c-4521-45f9-9dd1-0fb88c6ffb71.png">

### Actual Behavior

- italics do not render, but subscript works:

<img width="624" alt="image" src="https://user-images.githubusercontent.com/430126/156658095-885b32dc-c779-45da-afce-ae229dcc5371.png">

**Comment by pdpinch** [comment]
If this is very hard to fix, and we find a studio solution for mathjax, that might be a better approach.

**Comment by Wassaf-Shahzad** [comment]
@pdpinch 
i just checked this this issue and it's  pretty much the same as https://github.com/mitodl/ocw-hugo-themes/issues/490
However in this case since we cant move to a newline/space, replacing underscores with asterisks fixes this issue.
I replaced underscores with asterisks locally  and got the following result
<img width="1068" alt="Screenshot 2022-03-04 at 2 24 34 PM" src="https://user-images.githubusercontent.com/87968618/156738047-844242ae-42e3-429d-b53f-4af6ce5c077c.png">

**Comment by pdpinch** [comment]
@alicewriteswrongs or @gumaerc what do you think we can do here?

**Comment by alicewriteswrongs** [comment]
the real solution I think would be to re-write these formulas with mathjax, I think without doing that we're going to keep playing whack-a-mole with things like this

**Comment by pdpinch** [comment]
Is there a way we could have ckeditor write out `*` instead of `_` when the italic text is in the middle of a "word" ?

**Comment by gumaerc** [comment]
For what it's worth, it does seem that there's a plugin for CKEditor for MathJax editing: https://ckeditor.com/docs/ckeditor4/latest/examples/mathjax.html

I did some testing on this and found that if I select some text in the middle of a word in CKEditor and make it italic, it puts underscores around the text in the markdown.  Upon pulling this back up in CKEditor, you'll see the literal underscores in the editor.  I did some digging on why this is and came across this Github issue:

https://github.com/ckeditor/ckeditor5/issues/2402

Basically, it seems like the reasoning behind why this behaves this way has to do with something that Github did with Github Flavored Markdown.  They decided that underscores in the middle of a word are most commonly used in things like variable names and decided that they shouldn't be interpreted as markdown formatting. I couldn't see a clear way to do this at the moment, but if we used asterisks for both bold and italic formatting, that should work fine.  In markdown, asterisks and underscores are interchangeable for both bold and italics and by definition simply define the level of "emphasis."  For example:

```
_italics_
__bold__
___bold and italics___

*italics*
**bold**
***bold and italics***
```

renders as:

_italics_
__bold__
___bold and italics___

*italics*
**bold**
***bold and italics***

However, with Github Flavored Markdown you can see that underscores in the middle of a word are iterpreted differently than asterisks:

```
italics_in_themiddle
bold__in__themiddle
bolditalics___in___themiddle

italics*in*themiddle
bold**in**themiddle
bolditalics***in***themiddle
```

italics_in_themiddle
bold__in__themiddle
bolditalics___in___themiddle

italics*in*themiddle
bold**in**themiddle
bolditalics***in***themiddle

So, given that the italics button in CKEditor puts in underscores by default, we should find a way to override this behavior so that it uses asterisks.  This will be useful not just for mathematical formulas like this, but simply for having the editor function as expected.  Nonetheless, I think for math formulas like this we should implement the MathJax plugin and just allow people to use that.

**Comment by ChristopherChudzicki** [comment]
**tl;dr** There is a significant amount (but perhaps not huge) amount of existing data that will continue to exhibit the original bad-underscore behavior until someone fixes the markdown. This CSV lists many examples. (The CSV is all instances of `_{{` in OCW.) https://docs.google.com/spreadsheets/d/1YR9TjoknAM_lP5zVfI6Wv25ynQ0Q6cij8TqxOoU0tDk/edit?usp=sharing

### Details 

Setting aside whether or not authors should be specifying italics..., **they need to be asterisks not underscores in order for some things to display**. To expand on what @gumaerc said above, here are some examples:

<img width="654" alt="Screen Shot 2022-03-24 at 2 46 50 PM" src="https://user-images.githubusercontent.com/9010790/159988665-831bf382-3426-41b7-a11e-3024348d13a8.png">

Some notes:
- 3 does not seem realistic, but 4 is, as we've seen 
- this sort of content can come out of ocw-to-hugo. It may never have interacted with studio
- **Delightfully:** Github-flavored markdowns DOES interpret 4 as italics: `ant _bear_{{< sup 2 >}}_ cat` --> "ant _bear_{{< sup 2 >}} cat" because GFM doesn't replace the shortcode with alphanumeric characters.

    **This is really useful to us:** It means that although the published content looks wrong, if you open it in studio and edit+save, it will be fixed.

    _Caveat: Won't work if the underscore has been escaped, which will be true if the content was edited Prior to Wassaf's PR. But that is only the case for one page, which someone can fix in the admin panel._

Here is an exhaustive list of all pages that have `_{{` in their markdown: https://docs.google.com/spreadsheets/d/1YR9TjoknAM_lP5zVfI6Wv25ynQ0Q6cij8TqxOoU0tDk/edit?usp=sharing

**Comment by ChristopherChudzicki** [comment]
Going to close this, resolved by https://github.com/mitodl/ocw-studio/pull/1147 and the new issue https://github.com/mitodl/ocw-studio/issues/1160
